Submitted By: DJ Lucas <dj AT linuxfromscratch DOT org>
Date: 2006-06-25
Initial Package Version: 6u1
Upstream Status: Not applicable
Origin: Self 
Description: Do not compile legacy gcc-2.9 compatible Mozilla plugin.

--- jdk-build-orig/deploy/make/plugin/adapter/Makefile	2007-02-10 15:22:15.000000000 -0600
+++ jdk-build/deploy/make/plugin/adapter/Makefile	2007-02-11 11:07:12.000000000 -0600
@@ -18,9 +18,9 @@
   SUBDIRS += ns4-adapter
 endif
 
-ifeq ($(PLATFORM), linux)
-  SUBDIRS += oji-gcc29-adapter
-endif
+#ifeq ($(PLATFORM), linux)
+#  SUBDIRS += oji-gcc29-adapter
+#endif
 
 all build clobber:: $(SUBDIRS)
 
--- jdk-build-orig/deploy/make/plugin/nscore/Makefile	2007-02-10 15:22:15.000000000 -0600
+++ jdk-build/deploy/make/plugin/nscore/Makefile	2007-02-11 13:25:50.000000000 -0600
@@ -18,9 +18,9 @@
   SUBDIRS += unix
 endif
 
-ifeq ($(PLATFORM), linux)
-  SUBDIRS += linux-gcc29
-endif
+#ifeq ($(PLATFORM), linux)
+#  SUBDIRS += linux-gcc29
+#endif
 
 all build clobber:: $(SUBDIRS)
 
